include/deal.II/sundials/kinsol.h

```diff
@@ -161,7 +161,9 @@ namespace SUNDIALS
    *
    * Specifying residual() allows the user to use Newton and Picard strategies
    * (i.e., $F(u)=0$ will be solved), while specifying iteration_function(), a
-   * fixed point iteration will be used (i.e., $G(u)=u$ will be solved).
+   * fixed point iteration will be used (i.e., $G(u)=u$ will be solved). An
+   * error will be thrown if iteration_function() is called for Picard or
```

Comment:
```suggestion
   * error will be thrown if iteration_function() is set for Picard or
```
is that what you mean?